routes/api/contacts.js

```diff
@@ -1,25 +1,93 @@
-const express = require('express')
+import express from "express";
+import contactsService from "../../models/contacts.js";
+import { HttpError } from "../../helpers/index.js";
+import Joi from "joi";
 
-const router = express.Router()
+const contactsAddSchema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
Schema should be in separated file. You should validate phone and email properly

Reply:
Fixed